src/layouts/Root/components/Footer.tsx

```diff
@@ -21,6 +21,14 @@ const StyledFooterItem = styled(Link)`
     color: inherit;
   }
 
+  ${SELECTOR_FOCUS_VISIBLE} {
+    /*
+      updates the default outline-color from
+      css-bedrock for the darker footer background
+    */
+    outline-color: currentcolor;
```

Comment:
This fix feels like it belongs in `css-components` in order to make the anchor reset more bullet-proof.

Reply:
Well, if we are in `:active` or `:hover` state, this would result in darker (blue-600+) focus indicators. Wouldn't this deviate from design intention?